packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.scss

```diff
@@ -0,0 +1,155 @@
+$icon-title-size: 2.4rem;
+$icon-modify-size: 1.2rem;
+
+@mixin input-text($weight) {
+	color: $black;
+	font-size: 1.4rem;
+	font-weight: $weight;
+}
+
+@mixin ellipsis-title() {
```

Comment:
or just `@mixin ellipsis() {`

Reply:
Tututu if no arguments you have to delete the parenthesis too !  ;)